Frontend/src/pages/events/index.jsx

```diff
@@ -8,12 +9,25 @@ import {
   TableHeaderCell,
   TableRow,
 } from 'semantic-ui-react'
+import getCompetitionWcif from '../../api/competition/get/get_competition_wcif'
 import { CompetitionContext } from '../../api/helper/context/competition_context'
+import { setMessage } from '../../ui/events/messages'
+import LoadingMessage from '../../ui/messages/loadingMessage'
 import styles from './index.module.scss'
 
 export default function Events() {
   const { competitionInfo } = useContext(CompetitionContext)
-  return (
+  const { isLoading, data: wcif } = useQuery({
+    queryKey: ['wcif', competitionInfo.id],
+    queryFn: () => getCompetitionWcif(competitionInfo.id),
+    retry: false,
+    onError: (err) => {
+      setMessage(err.message, 'error')
+    },
+  })
+  return isLoading ? (
+    <LoadingMessage />
+  ) : (
```

Comment:
good point, while we display an error message it will error again when it tries to load this. I guess if it errors it should redirect somewhere?

Reply:
I don't remember the context enough, and the site I have bookmarked gives a 503 when I try to check. But if this component is integral to the functionality of the page it's on then I would guess showing some generic error in it's place, and asking the user to refresh or try again later, would make the most sense?

Reply:
yes the site currently 503s because I am moving it to real staging with this PR!. 
I am already showing an error when the fetch fails, so I think we can just not load the component

Reply:
Added a `<Message>Loading (Events|Schedule) failed, please try again</Message>`